docs/modules/ROOT/pages/tutorials/install.adoc

```diff
@@ -0,0 +1,171 @@
+For this to work `k8up` needs to be installed on the same cluster using the https://github.com/projectsyn/component-backup-k8up[k8up component].
+
+By default the component uses the following configuration:
+
+[source,yaml]
+----
+backup:
+  enabled: true
+  schedule: '*/13 * * * *'
+  keepjobs: 5
+  password: '?{vaultkv:${customer:name}/${cluster:name}/vault/${_instance}/backup/password}'
+  bucket:
+    name: '${_instance}-backup'
+    accesskey: '?{vaultkv:${customer:name}/${cluster:name}/vault/${_instance}/backup/s3_access_key}'
+    secretkey: '?{vaultkv:${customer:name}/${cluster:name}/vault/${_instance}/backup/s3_secret_key}'
+----
+
+This means it expects that there is a bucket with the name `+${INSTANCE}-backup+` and that three secrets need to be added to the deploying Vault.
+
+* The password to use for backup encryption to the key `+clusters/kv/${TENANT_ID}/${CLUSTER_ID}/vault/${INSTANCE}/backup/password}+`
+* The access key for the bucket to to the key `+clusters/kv/${TENANT_ID}/${CLUSTER_ID}/vault/${INSTANCE}/backup/s3_access_key}+`
+* The secret key for the bucket to to the key `+clusters/kv/${TENANT_ID}/${CLUSTER_ID}/vault/${INSTANCE}/backup/s3_secret_key}+`
+
+If you only deploy a single vault to the cluster, `+${INSTANCE}+` will simply be `+vault+`.
+See <<Multiple Instances>> for more details on this parameter.
+
+== LDAP
+
+By default, only the backup service account and the root user has access to vault.
+You have the option to add more authentication methods and policies. 
+Here is an example to add vault access to all people in an LDAP group.
+
+[source,yaml]
+----
+config:
+  auth:
+    - type: ldap
+      description: LDAP auth
+      options:
+        listing_visibility: "unauth"
+      config:
+        url: ldaps://ldap.server.net:636
+        binddn: "uid=vault-service,ou=Special Users,dc=server,dc=net"
+        bindpass: '?{vaultkv:${customer:name}/${cluster:name}/vault/${_instance}/ldappass}'
+        userattr: uid
+        userdn: "ou=vault,ou=Service Access,ou=Views,dc=server,dc=net"
+        groupdn: "ou=Groups,dc=vshn,dc=net"
+        groupattr: cn
+      groups:
+        vault-root:
+          policies: cluster-root
+  policies:
+    - name: cluster-root
+      rules: |
+        path "clusters/*" {
+          capabilities = ["create", "read", "update", "delete", "list"]
+        }
+----
+
+Under the hood, this component uses `+bank-vaults+` to configure vault.
+https://banzaicloud.com/docs/bank-vaults/external-configuration/[This reference] provides more details on how to configure policies and authentication methods.
+
+== Lieutenant Integration
+
+If vault interacts with a Lieutenant instance, we need to add some policies and authentication methods.
+
+[source,yaml]
+----
+config:
+  policies:
+    - name: syn-cluster
+      rules: | # Allow to get cluster's own secrets
+        path "clusters/kv/data/+/{{identity.entity.aliases.auth_kubernetes_5b636428.metadata.service_account_name}}/*" {
```

Comment:
```suggestion
        path "clusters/kv/data/+/{{identity.entity.aliases.auth_kubernetes_5b636428.metadata.service_account_name}}/*" {
```

Probably need to make a note that the `auth_kubernetes_<id>` part of this path needs to be identified after activating Kubernetes auth in Vault.

Also note that this config expects that Vault and Lieutenant are deployed in the same cluster.